gemini_proact_flutter/lib/view/root.dart

```diff
@@ -16,7 +17,10 @@ class Proact extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
       ),
-      home: AuthPage() // returns either HomePage or LoginPage based on user auth status
+      home: Scaffold(
+        body: OnboardingForm()
```

Comment:
- [ ] This will make it difficult to integrate with other changes in #54. 
I will try to make them compatible with each other.